template.hbs

```diff
@@ -162,7 +166,9 @@
               <li>{{get Certificate.CommercialTransaction "A06.1,ZipCode"}},
                 {{get Certificate.CommercialTransaction "A06.1,City"}},
                 {{get Certificate.CommercialTransaction "A06.1,Country"}}</li>
+              {{#if (get Certificate.CommercialTransaction "A06.1,Email" false)}}
```

Comment:
as a reminder that requires to provide a new `get` HBS helper and/or to update schema-tools-generate-html provided HBS helpers.

Reply:
Thanks for remindming me. I did the change only locally.

Reply:
I updated the `get` helper in this [PR](https://github.com/s1seven/schema-tools/pull/157)